airflow/models/taskinstance.py

```diff
@@ -3529,7 +3543,7 @@ def from_dict(cls, obj_dict: dict) -> SimpleTaskInstance:
         return cls(**obj_dict, start_date=start_date, end_date=end_date, key=ti_key)
 
 
-class TaskInstanceNote(Base):
+class TaskInstanceNote(TaskInstanceDependencies):
```

Comment:
Same here (even if the map index doesn't have a default, I guess it should be ok)